sunpy/map/tests/test_maputils.py

```diff
@@ -277,38 +275,6 @@ def test_contains_coord(aia171_test_map):
     assert (contains_coordinate(smap, multi_coord) == [True, False]).all()
 
 
-def test_extract_along_coord(aia171_test_map):
-    nmax = max(*aia171_test_map.data.shape)
-    top_right = aia171_test_map.pixel_to_world((nmax-1)*u.pix, (nmax-1)*u.pix)
-    line = SkyCoord([aia171_test_map.bottom_left_coord, top_right])
-    with pytest.warns(SunpyDeprecationWarning):
-        intensity, line_discrete = extract_along_coord(aia171_test_map, line)
-    pix_diag = np.array([(i, i) for i in range(nmax)])
-    intensity_diag = u.Quantity(
-        [aia171_test_map.data[i[0], i[1]] for i in pix_diag],
-        aia171_test_map.unit
-    )
-    line_discrete_pix = aia171_test_map.world_to_pixel(line_discrete)
-    assert np.allclose(pix_diag[:, 0], line_discrete_pix.x.value)
-    assert np.allclose(pix_diag[:, 1], line_discrete_pix.y.value)
-    assert u.quantity.allclose(intensity_diag, intensity)
-
-
-@pytest.mark.filterwarnings("ignore:The extract_along_coord function is deprecated:sunpy.util.exceptions.SunpyDeprecationWarning")
-def test_extract_along_coord_one_point_exception(aia171_test_map):
```

Comment:
just looking at this, this should really be added as a test to `pixelate_coord_path`, and similarly with the test below. Maybe they can go in a different pr  though

Reply:
If they go into a separate PR, we can backport it easier. Do we want to track this with an issue?

Reply:
Why would we need to backport anything here?

Reply:
If we want to migrate these tests to the current API, I think those should be backported to ensure we don't break anything in the future.

Reply:
could they just be added in a new PR? (also the second test is something that is needed for the pixelate_coord_path I think (see #7205)

Reply:
Aside from the fix that is needed for #7205, these tests do not need to be preserved.